pkg/controller/controller.go

```diff
@@ -85,6 +87,16 @@ func (s *Service) Init() error {
 		s.isHealthMonitorEnabled, _ = strconv.ParseBool(isHealthMonitorEnabled)
 	}
 
+	if nfsAcls, ok := csictx.LookupEnv(ctx, common.EnvNfsAcls); ok {
```

Comment:
Can this loop be simplified instead of having this many conditions can we just assign `nfsAcls = fmt.Sprintf("%s", os.ModePerm)`  directly and upon any different value is available we can overwrite the default value.

Reply:
Good point. done, actually setting to default value is done in stager.go, so removed it here.